packages/core/src/ceramic.ts

```diff
@@ -922,6 +952,7 @@ export class Ceramic implements CeramicApi {
    */
   async close(): Promise<void> {
     this._logger.imp('Closing Ceramic instance')
+    await this.syncApi.shutdown()
```

Comment:
Does the sync API need to be shutdown first please?
If not, maybe the synchronous calls can be done first, then the async ones? And possibly in parallel if order doesn't matter?